kfs-core/src/main/java/edu/arizona/kfs/sys/KFSPropertyConstants.java

```diff
@@ -26,6 +26,7 @@
 
     public static final String GEC_DOCUMENT_NUMBER = "gecDocumentNumber";
     public static final String ENTRY_ID = "entryId";
+    public static final String CARD_CYCLE_VOL_LIMIT = "cardCycleVolLimit";

```

Comment:
A reference check here isn't showing anything else using this -- is my IDE confused?